charts/digital-product-pass/templates/ingress-backend.yaml

```diff
@@ -1,26 +1,26 @@
 #################################################################################
 # Tractus-X - Digital Product Passport Application
-  #
-  # Copyright (c) 2022, 2024 BMW AG, Henkel AG & Co. KGaA
```

Comment:
There should be only year of creation in the license headers. BMW and Henkel should be in two different lines.
The updated license header should look like this:

Copyright (c) 2022 BMW AG,
Copyright (c) 2022 Henkel AG & Co. KGaA
Copyright (c) 2023 CGI Deutschland B.V. & Co. KG